oak_idl_gen_services/src/lib.rs

```diff
@@ -245,15 +245,15 @@ fn generate_client_method(rpc_call: &RPCCall) -> anyhow::Result<Vec<String>> {
             response_type(rpc_call)
         ),
         format!(
-            "        flatbuffers::root::<{}>(request_body).map_err(|_err| oak_idl::Error::InvalidRequest)?;",
+            "        flatbuffers::root::<{}>(request_body).map_err(|_err| oak_idl::Error {{ code: oak_idl::ErrorCode::InvalidRequest, message: None }})?;",
```

Comment:
I think implementing `Error::new` that takes just a code and sets an empty message, and `Error::new_with_message` will make this a bit cleaner.

Also, should the original error not be printed into the message, rather than ignoring it?

Reply:
thanks, addressed